lib/routes/transformer-circuits/index.ts

```diff
@@ -43,37 +47,81 @@ async function handler() {
                 // Extract title
                 const title = currentElement.find('h3').text().trim();
 
-                // Extract author if available (papers have bylines, notes don't always have them)
+                // Extract author if available
                 let author = '';
                 const byline = currentElement.find('.byline');
                 if (byline.length) {
                     author = byline.text().trim();
                 }
+
                 // Extract description
                 const description = currentElement.find('.description').text().trim();
 
                 // Get the article URL
                 const href = currentElement.attr('href');
                 const articleUrl = href ? (href.startsWith('http') ? href : `${rootUrl}/${href}`) : rootUrl;
 
-                // Create article object
+                // 使用 cache.tryGet 缓存文章内容
+                const fullContent = await cache.tryGet(
+                    articleUrl,
+                    async () => await fetchArticleContent(articleUrl),
+                    60 * 60 * 24 // 缓存24小时
```

Comment:
Do not hard code the cache duration. It should be managed by https://github.com/DIYgod/RSSHub/blob/5b58a1911e0644c4c0130a795d1ed9c2d82678d6/lib/config.ts#L465